include/yaramod/types/expressions.h

```diff
@@ -1325,6 +1380,10 @@ class ArrayAccessExpression : public IdExpression
 	const Expression::Ptr& getArray() const { return _array; }
 	const Expression::Ptr& getAccessor() const { return _accessor; }
 
+	//TODO: check if is correct:
```

Comment:
Should this still be here?

Reply:
No, this can be removed. I am sorry I ignored all those `TODO`. I removed it and it should be fine - thank you.

Reply:
There should be no more TODOs in yaramod (in our code) now.